migration/load/sources.csv

```diff
@@ -1,7 +1,7 @@
 "Source label","Database table / API endpoint","Data point","Source filename","Source sheet","Source range","Author","Date"
 "B.C. Labour Market Information Office, derived from 2022 Job Bank Wage data","wages",,"WorkBC_2022_Wage_Data - UPDATED June 28 2023.xlsx","Sheet1","A2-G501",,"2023/06/28"
 ,"titles",,"Common Job Titles revised - July 18 2017 (RA).xlsx","Sheet1","A2-D7490",,
-,"skills",,"UPDATED FINAL Skills Data for Career Profiles (updated April16 19).xlsx","Sheet1","A2-F17396",,
+,"skills",,"UPDATED FINAL Skills Data for Career Profiles (updated Sept 12 23).xlsx","Sheet1","A2-F17396",,
```

Comment:
please add "2023/09/12" as the last value of this row to indicate the date of the dataset.

Reply:
and reexport the dump, of course.